docs/uk/docs/external-links.md

```diff
@@ -0,0 +1,82 @@
+# Зовнішні посилання та статті
+
+**FastAPI** має велику спільноту, яка постійно зростає.
+
+Існує багато публікацій, статей, інструментів та проектів, пов’язаних із **FastAPI**.
+
+Ось неповний список деяких з них.
+
+!!! підказка
+    Якщо у вас є стаття, проект, інструмент або що-небудь інше, що стосується **FastAPI**, та ще не зазначеного тут, створіть <a href="https://github.com/tiangolo/fastapi/edit/master/docs/en/data/external_links.yml" class="external-link" target="_blank">пулл реквест</a>.
+
+## Статті
+
+### Англійською
+
+{% if external_links %}
+{% for article in external_links.articles.english %}
+
+* <a href="{{ article.link }}" class="external-link" target="_blank">{{ article.title }}</a> by <a href="{{ article.author_link }}" class="external-link" target="_blank">{{ article.author }}</a>.
+{% endfor %}
+{% endif %}
+
+### Японською
+
+{% if external_links %}
+{% for article in external_links.articles.japanese %}
+
+* <a href="{{ article.link }}" class="external-link" target="_blank">{{ article.title }}</a> by <a href="{{ article.author_link }}" class="external-link" target="_blank">{{ article.author }}</a>.
```

Comment:
I was wondering if "title **by** author" should be translated... 
Seems to be a convention though, see e.g. Polish https://fastapi.tiangolo.com/pl/external-links/

Reply:
Probably wouldn't hurt